app/screens/post_options/post_options.js

```diff
@@ -287,6 +272,41 @@ export default class PostOptions extends PureComponent {
         actions.addReaction(post.id, emoji);
     };
 
+    handleReply = () => {
+        this.goToThread();
```

Comment:
`goToThread` is not needed, you can just use this function and include the code in the `goToThread` one

Reply:
Okie doke :)